addons/website_forum/tests/test_performance.py

```diff
@@ -62,6 +62,13 @@ def test_10_perf_sql_forum_standard_data(self):
         self.assertLessEqual(n, 26)
         n = self._get_url_hot_query(self.post.website_url, cache=False)
         self.assertLessEqual(n, 28)
+        # Without microdata shouldn't change the score.
+        n = self._get_url_hot_query(self.post.website_url + '?no_microdata=1')
+        self.assertLessEqual(n, 26)
+        # It appears that the 'cache' parameter fails to function correctly when appended to a URL
+        # already containing parameters.
+        n = self._get_url_hot_query(self.post.website_url + '?no_microdata=1&debug=disable-t-cache')
+        self.assertLessEqual(n, 28)
```

Comment:
You don't need this no_microdata stuff. The perf are the same before and after your imp. It's ok like that :+1: